src/bucket/BucketManager.cpp

```diff
@@ -1504,18 +1504,18 @@ visitLiveEntriesInBucket(
             if (e.type() == LIVEENTRY || e.type() == INITENTRY)
             {
                 auto const& liveEntry = e.liveEntry();
-                if (minLedger && liveEntry.lastModifiedLedgerSeq < *minLedger)
+                if (!processedEntries
```

Comment:
Just to double check my understanding, is the bug here we would sometimes stop the scan prematurely due to a shadowed `lastModifiedLedgerSeq` value that shouldn't actually be counted?